### ISSUE TITLE
Fix IntervalArray docstrings with missing period in the summary

### ISSUE THREAD

**Issue by datapythonista** [python-sprints/pandas-mentoring#135]
Same as #132, but for other docstrings:

```
pandas.arrays.IntervalArray.left: Summary does not end with a period
pandas.arrays.IntervalArray.right: Summary does not end with a period
pandas.arrays.IntervalArray.closed: Summary does not end with a period
pandas.arrays.IntervalArray.mid: Summary does not end with a period
pandas.arrays.IntervalArray.length: Summary does not end with a period
pandas.arrays.IntervalArray.is_non_overlapping_monotonic: Summary does not end with a period
pandas.arrays.IntervalArray.from_tuples: Summary does not end with a period
pandas.arrays.IntervalArray.set_closed: Summary does not end with a period
pandas.arrays.IntervalArray.to_tuples: Summary does not end with a period
```

**Comment by sparalic** [comment]
@datapythonista  I will take up this one

**Comment by sparalic** [comment]
@datapythonista it's complete.

**Comment by datapythonista** [comment]
great, thanks, you can close this issue once the PR is merged.

**Comment by sparalic** [comment]
will do!

**Comment by datapythonista** [comment]
@sparalic you can also check this notebook that @TanyaaCJain created: https://github.com/python-sprints/pandas-mentoring/blob/master/notebooks/docstring_error_frequency.ipynb

The histogram at the end are all errors that we can help fix. We are now working on `SS03`, one of the somehow small bars. :)

**Comment by sparalic** [comment]
@datapythonista great, I will look for others as well.

**Comment by sparalic** [comment]
closed, merged in pandas #28043
